modules/news/news.model.js

```diff
@@ -0,0 +1,18 @@
+const newsType = `
+_id: ID!
+title: [Language!]
+text: [Language!]
+images: [PrimaryImage!]
+video: String!
```

Comment:
video is not required field but every article(news) should have Image for a preview page

Reply:
fixed